lib/src/geojson.dart

```diff
@@ -317,12 +321,30 @@ class BBox extends CoordinateType {
   bool get _is3D => length == 6;
 
   num get lng1 => _items[0];
+
   num get lat1 => _items[1];
+
   num? get alt1 => _is3D ? _items[2] : null;
+
   num get lng2 => _items[_is3D ? 3 : 2];
+
   num get lat2 => _items[_is3D ? 4 : 3];
+
   num? get alt2 => _is3D ? _items[5] : null;
 
+  BBox copyWith({
+    num? lng1,
+    num? lat1,
+    num? lng2,
+    num? lat2,
+  }) =>
+      BBox(
+        lng1 ?? this.lng1,
+        lat1 ?? this.lat1,
+        lng2 ?? this.lng2,
+        lat2 ?? this.lat2,
```

Comment:
what about the altitudes alt1, alt2?

Reply:
can be fixed later. go ahead and merge!

Reply:
I added those as well. But I assume that those should be done in other places too.

Reply:
But you did it after merge?

Reply:
Yes, after merging. 
We need a PR